public/app/features/alerting/unified/mocks/silences.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpResponse, http } from 'msw';
+
+import { mockSilences } from 'app/features/alerting/unified/mocks';
+
+//////////////
+// Silences //
+//////////////
+
+export const silencesListHandler = (silences = mockSilences) =>
+  http.get('/api/alertmanager/:datasourceUid/api/v2/silences', () => HttpResponse.json(silences));
+
+export const silenceGetHandler = () =>
+  http.get<{ uuid: string }>('/api/alertmanager/:datasourceUid/api/v2/silence/:uuid', ({ params }) => {
+    const { uuid } = params;
+    const matchingMockSilence = mockSilences.find((silence) => silence.id === uuid);
```

Comment:
How to override the default silences?

Reply:
For now, all we need in order to be able to test is either fetching an existing one, or trying to get a non-existent one. 

So I haven't added in a method to configure and return a different one, but if we need to test another case, we just add it to the array of silences that we mock, make the ID of that silence a `const`, and use that in our tests as well. What do you think?